grzyClothTool/Controls/Drawable/DrawableList.xaml.cs

```diff
@@ -56,5 +61,79 @@ private void OpenFileLocation_Click(object sender, RoutedEventArgs e)
             var drawable = DrawableListSelectedValue as GDrawable;
             FileHelper.OpenFileLocation(drawable.FilePath);
         }
+
+        private void RemoveDrawable_Click(object sender, RoutedEventArgs e)
+        {
+            var drawable = DrawableListSelectedValue as GDrawable;
+            MainWindow.AddonManager.DeleteDrawable(drawable);
+        }
+
+        private void ReplaceDrawable_Click(object sender, RoutedEventArgs e)
+        {
+            var drawable = DrawableListSelectedValue as GDrawable;
+
+            OpenFileDialog files = new()
+            {
+                Title = $"Select drawable file to replace '{drawable.Name}'",
+                Filter = "Drawable files (*.ydd)|*.ydd",
+                Multiselect = false
+            };
+
+            if (files.ShowDialog() == true)
+            {
+                drawable.FilePath = files.FileName; // changing just path - might need to be updated to CreateDrawableAsync
+                SaveHelper.SetUnsavedChanges(true);
+
+                CWHelper.SendDrawableUpdateToPreview(e);
+            }
+        }
+
+        async private void ExportDrawable_Click(object sender, RoutedEventArgs e)
+        {
+            var drawable = DrawableListSelectedValue as GDrawable;
+
+            MenuItem menuItem = sender as MenuItem;
+            var tag = menuItem?.Tag?.ToString();
+
+
+            // pretty ugly way of setting title, might imrpove later
+            OpenFolderDialog folder = new()
+            {
+                Title = tag == null ? "Select the folder to export drawable" :
+                        tag == "YTD" ? "Select the folder to export drawable with textures" :
+                        $"Select the folder to export textures as {tag}",
+                Multiselect = false
+            };
+
+            if (folder.ShowDialog() != true)
+            {
+                return;
+            }
+
+            string folderPath = folder.FolderName;
+
+            try
+            {
+                // Export just textures
+                // this is the only case we don't export drawable, so return
+                if (tag == "DDS" || tag == "PNG") 
+                {
+                    await Task.Run(() => FileHelper.SaveTexturesAsync(new List<GTexture>(drawable.Textures), folderPath, tag));
+                    return;
```

Comment:
I think this can be simplified

```
try
{
	if (!string.IsNullOrEmpty(tag) && (tag == "YTD" || tag == "PNG" || tag == "DDS")) 
	{
		await FileHelper.SaveTexturesAsync([drawable.Textures], folderPath, tag);
		
		if (tag == "DDS" || tag == "PNG") 
		{
			return;
		}

	}
	
	await FileHelper.SaveDrawablesAsync([drawable], folderPath);
}
```

Reply:
btw. for all I/O actions (copy  files, save to PC etc.) it's better to await instead of wrapping in Task.Run(() => ....)